nodejs/week2/meal_sharing_website/src/backend/routes/meals.js

```diff
@@ -0,0 +1,42 @@
+const express = require("express");
+const router = express.Router();
+const meals = require("../data/meals.json");
+
+//Respond with the json for the meal with the corresponding id
+router.get("/:id", (req, res) => {
+	const mealsId = parseInt(req.params.id);
+	const mealsIdOutput = meals.filter((item) => item.id == mealsId);
+	mealsIdOutput.length > 0 ? res.json(mealsIdOutput) : res.status(404).send(`Error: Unable to fetch the request`);
+});
+
+//http://localhost:8080/meals/1
+
+//Get meals that has a price smaller than maxPrice
+router.get("/", (req, res) => {
+	let meal;
+	const maxPrice = parseFloat(req.query.maxPrice);
+	const title = req.query.title;
+	const date = parseInt(req.query.createdAfter);
+	const limit = parseInt(req.query.limit);
+
+	if (maxPrice >= 0) {
```

Comment:
It is not necessity to compare maxPrice with 0. You can just check if it is exists.
Like i`f (maxPrice) {`

the same for limit.